frontend/app/admin/src/views/organizations/EditOrganizationView.vue

```diff
@@ -120,7 +130,30 @@ const active = computed({
     set: value => addPatch({ active: value }),
 });
 
-const tags = computed(() => platform.value.config.tags);
+const platformTags = computed(() => platform.value.config.tags);
+const rootTags = computed(() => TagHelper.getRootTags(platformTags.value));
+const selectedTagIds = computed(() => patched.value.meta.tags);
+const selectedTags = computed(() => tagIdsToTags(selectedTagIds.value));
+const allTagsWithChildren = computed(() => platformTags.value.filter(tag => tag.childTags.length > 0));
+const selectedTagsWithChildren = computed(() => selectedTags.value.filter(tag => tag.childTags.length > 0));
+
+watch(() => selectedTagsWithChildren.value, (selectedTagsWithChildren) => {
+    const tagIds = selectedTagIds.value;
+
+    // delete tags that are subtags of tags that are not selected
+    const tagsToDelete = tagIds.filter(tagId =>
+        !(rootTags.value.some(rootTag => rootTag.id === tagId)
+            || selectedTagsWithChildren.some(tagWithChildren => tagWithChildren.childTags.includes(tagId))
+        ));
+
+    if (tagsToDelete.length > 0) {
+        setTagIds(tagIds.filter(id => !tagsToDelete.includes(id)));
+    }
+}, { immediate: true });
```

Comment:
Als je root tags hebt, en die later verplaatst onder een bestaande tag (kan voorlopig niet via de UI), dan wil je niet dat we die tags automatisch verwijderen - we zouden in plaats daarvan automatisch de root tags moeten toevoegen, maar dit zou gewoon in de backend afgehandeld moeten worden denk ik. Lijkt me risicovol met zo een watcher